libr/anal/flirt.c

```diff
@@ -767,25 +760,25 @@ static ut8 read_module_referenced_functions(RFlirtModule *module, RBuffer *b) {
 		if (!ref_function) {
 			goto err_exit;
 		}
-		if (version >= 9) {
+		if (b->version >= 9) {
```

Comment:
This version is about the flirt header and it have nothing to do with rbuffer. Maybe better to hold a struct with those vars and also store rbuffer inside and pass it to all the functions instead of adding unrelated vars to rbuffer